docker-compose.production.yml

```diff
@@ -82,6 +82,16 @@ services:
     secrets:
       - petabox_seed
 
+  solr-updater:
+    restart: always
+    hostname: "$HOSTNAME"
+    environment:
+      - OL_CONFIG=/olsystem/etc/openlibrary.yml
+      - PYENV_VERSION=3.8.6
```

Comment:
Copy any other parameters from https://github.com/internetarchive/olsystem/blob/master/etc/supervisor/conf.d/solr-updater.conf

Reply:
Do you mean lines 5 and 6 or lines 7 thru 11?